rfc/5/index.md

```diff
@@ -743,7 +693,126 @@ Practically, non-invertible transformations have finite extents,
 so bijection transforms should only be expected to be correct / consistent for points that fall within those extents.
 It may not be correct for any point of appropriate dimensionality.
 
+### "scene" metadata
+
+For images that share a spatial relationship,
+the "scene" metadata layout can be used to describe the relationship between images.
+The "scene" dictionary is located under the custom attributes of a parent-level zarr-group
+that contains the related images as child groups.
+
+The "scene" dictionary MUST contain the field `coordinateTransformations`,
+whose value MUST be a list of valid [transformations](transformation-types).
+It MAY contain the field `coordinateSystems`,
+whose value MUST be a list of valid [coordinate systems](#coordinatesystems-metadata).
+
+If used inside "scene" metadata, the `input` and `output` fields of `coordinateTransformations` MUST contain a json object,
+which MUST contain either the `path` or the `name` field, or both.
+The value of the `path` field is the path to a multiscale image group in the same container.
```

Comment:
What is the intended meaning of an object like `{"path": "path/to/some/object"}`? As `"multiscales":` is an array, a path to a group containing multiscale metadata isn't sufficient to identify a coordinate system. Should this be "path to a zarr array referenced by multiscale metadata"? Then the absence of `"name"` would imply the intrinsic/ default array coordinate system.